.gitignore

```diff
@@ -15,4 +15,7 @@ public/app.js
 
 # Keys
 apiKeys.js
-apikeys.js
```

Comment:
We should keep the apikeys.js file in gitignore so that our API keys are not shared on github.